python/lsst/meas/extensions/scarlet/source.py

```diff
@@ -33,9 +33,46 @@
 logger = lsst.log.Log.getLogger("meas.deblender.deblend")
 
 
+def hasEdgeFlux(source, edgeDistance=1):
+    """hasEdgeFlux
+
+    Determine whether or not a source has flux within `edgeDistance`
+    of the edge.
```

Comment:
Are these the edges of a box containing the entire footprint to deblend, or a box only containing an individual source, i.e. a smaller view?

Reply:
Good question. `source.get_model` returns an array that is the size of the blend, with the source injected into it. So this only tests against a true image edge.